Server/mods/deathmatch/logic/CPed.h

```diff
@@ -103,6 +103,41 @@ enum eBone
     BONE_RIGHTFOOT
 };
 
+struct SPlayerAnimData
+{
+    std::string blockName;
+    std::string animName;
+    int         time{-1};
+    bool        loop{true};
+    bool        updatePosition{true};
+    bool        interruptable{true};
+    bool        freezeLastFrame{true};
+    int         blendTime{250};
+    bool        taskToBeRestoredOnAnimEnd{false};
+
+    std::int64_t startedTick{0};
+
+    float progress{0.0f};
+    float speed{1.0f};
+
+    SPlayerAnimData() = default;
+
+    SPlayerAnimData(const std::string& block, const std::string& anim, int time, bool loop, bool updatePos, bool interrupt, bool freeze, int blend,
+                    bool taskRestore, std::int64_t tick)
+        : blockName(block),
+          animName(anim),
+          time(time),
+          loop(loop),
+          updatePosition(updatePos),
+          interruptable(interrupt),
+          freezeLastFrame(freeze),
+          blendTime(blend),
+          taskToBeRestoredOnAnimEnd(taskRestore),
+          startedTick(tick),
+          progress(0.0f),
+          speed(1.0f){};
```

Comment:
Just terrible to whoever is going to read the code in the future. Remove this constructor and assign the fields directly. You also don't have to assign `progress` and `speed` with the same value.